librz/type/base.c

```diff
@@ -81,6 +81,8 @@ static RzBaseType *get_enum_type(RzTypeDB *typedb, const char *sname) {
 	}
 	free(members);
 
+	base_type->size = base_type->enum_data.cases.len * base_type->enum_data.cases.elem_size;
```

Comment:
It doesn't work like that. Enum size is either `uint32_t`  or `uint64_t` depending on how it's defined.